internal/handlers/telegram/handler_test.go

```diff
@@ -130,3 +132,122 @@ func TestJoinNoUsername(t *testing.T) {
 	}
 	joinHandler(clientObj, testListUser)
 }
+
+func TestStickerSmile_withusername(t *testing.T) {
+	testUser := &tgbotapi.User{
+		ID:        1,
+		UserName:  "test",
+		FirstName: "testing",
+		LastName:  "123",
+	}
+	correct := fmt.Sprintf("<%s (@%s)> 😄", testUser.FirstName, testUser.UserName)
```

Comment:
I would prefer we keep to only grabbing the username when sending messages in their name.
`<fmt.Sprintf("<%s> 😄", testUser.UserName)`

Reply:
fixed